src/components/QuestionerButton/QuestionerButton.jsx

```diff
@@ -0,0 +1,37 @@
+import "./QuestionerButton.css";
+import { useNavigate } from "react-router-dom";
+
+const QuestionerButton = ({
+  isFirstQuestion,
+  isLastQuestion,
+  nextRoute,
+  Finish,
```

Comment:
Unused prop, let's remove it

Reply:
I removed the finish props, if I remove the rest the app will crash

Reply:
You can also remove the `startRoute` prop from this component.